core/src/main/java/org/testcontainers/containers/Container.java

```diff
@@ -32,6 +32,11 @@ default SELF self() {
         return (SELF) this;
     }
 
+    @Override
+    default String getContainerName() {
```

Comment:
`HostPortWaitStrategy` uses `getContainerName()`, which was previously protected visibility in `GenericContainer`. 
It's only used for a debug statement, so it could probably be removed if you think this method shouldn't be added.

Reply:
since the method was protected, this is a breaking change anyway (making it public)
Let's just not expose this method and use `getContainerInfo().getName()` in `HostPortWaitStrategy`, or maybe even remove it since the name is not that helpful anyway (we assign random names)